keyboards/aleblazer/zodiark/info.json

```diff
@@ -13,6 +13,7 @@
     },
     "rgblight": {
         "max_brightness": 170,
+        "split": true,
```

Comment:
`RGBLED_SPLIT` unconditionally configures `RGBLIGHT_SPLIT` (quantum/rgblight/rgblight_post_config.h) so migrating to both "rgblight.split" and "rgblight.split_count" is somewhat pointless when it can be more minimal with just  `rgblight.split_count`

Reply:
What's the point of this define then?

I also note there are a few boards which have `rgblight.split` but not `split_count`:
```
draculad
handwired/brain
hidtech/bastyl
jian/nsrev2
jian/rev1
jian/rev2
kagizaraya/scythe
mint60
redox/rev1
```

Reply:
When specified without `RGBLED_SPLIT`, the effects basically mirror on each hand.